Dockerfile

```diff
@@ -2,15 +2,9 @@ FROM xabush/annotation-scheme:0.1
 MAINTAINER Abdulrahman Semrie
 
 WORKDIR $HOME
-RUN git clone https://git.savannah.nongnu.org/git/nyacc.git && \
-    cd nyacc && \
-    ./configure && \
-    make && \
-    make install
-
-WORKDIR $HOME
-RUN git clone https://github.com/Habush/guile-json.git && \
+RUN git clone https://github.com/aconchillo/guile-json && \
     cd guile-json && \
+    git reset --hard 1.2.0 && \
```

Comment:
BTW does it really need this specific version 1.2.0, I accidentally pulled the latest 3.3.0, and it seems to work too?